mne/viz/tests/test_epochs.py

```diff
@@ -79,25 +79,14 @@ def test_plot_epochs(capsys):
     fig = epochs[0].plot(picks=[0, 2, 3], scalings=None)
     fig.canvas.key_press_event('escape')
     plt.close('all')
+    keystotest = ['b', 'b', 'left', 'right', 'up', 'down',
```

Comment:
why testing `b` twice? is it to enter and then exit butterfly mode?

Reply:
I think so, yes

Reply:
Exactly.